runtime/bcutil/cfreader.c

```diff
@@ -2402,12 +2403,19 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 	NEXT_U16(classfile->minorVersion, index);
 	NEXT_U16(classfile->majorVersion, index);
 
+	/* Ensure that this is a supported class file version. */
+	if(checkClassVersion(classfile, segment, vmVersionShifted)) {
```

Comment:
Can you fix the formatting of the if statement to add a space between `if (` since we're touching the code anyway?